src/proxy-ui-api/frontend/scripts/dtsgen-wrapper.ts

```diff
@@ -4,11 +4,11 @@ import {parseFileContent} from 'dtsgenerator/dist/utils';
 import prettier from 'prettier';
 
 const openAPIPath = '../src/main/resources/openapi-definition.yaml';
-const outputTSPath = 'src/types.ts';
+const outputTSPath = 'src/openapi-openapi-types.ts';
```

Comment:
This seems to output a file called `openapi-openapi-types.ts` but `@/openapi-types` is still being imported inside the .vue files.

Reply:
I guess this is due to somekind of copy-paste typo. 

Fixed.